readthedocs/search/views.py

```diff
@@ -19,6 +20,15 @@
 log = logging.getLogger(__name__)
 LOG_TEMPLATE = '(Elastic Search) [%(user)s:%(type)s] [%(project)s:%(version)s:%(language)s] %(msg)s'
 
+PAGE_SIZE = 9
```

Comment:
Se questi sono i risultati mostrati per pagina, facciamo 12.

Reply:
qui secondo me : 
* lo esponiamo come setting
* è un parametro passato in querystring con un default (e un massimo)

Reply:
Non avendo altre specifiche se non il documento collegato all'epica ho usato 9.

Per ora aggiorno a 12 e ragionerei in una issue a parte se vogliamo/possiamo rendere il valore parametrico in base alla grafica.

Reply:
Ho creato #401 per tracciare questo miglioramento